src/components/Filters/AllyFilter.jsx

```diff
@@ -91,7 +91,7 @@ function BusinessFilter(props) {
             value={location}
             id="location"
             type="text"
-            placeholder="e.g. 80219"
+            placeholder="Denver or 80219"
```

Comment:
```suggestion
            placeholder="e.g. 80219"
```

We're not allowing filter by city name on Allies for v0, this was a last minute thing :(